dm/syncer/syncer.go

```diff
@@ -3349,26 +3345,34 @@ func (s *Syncer) Close() {
 	s.stopSync()
 	s.closeDBs()
 
+	// try to rollback checkpoints, if they already flushed, no effect, this operation should call before close schemaTracker
```

Comment:
pause task will not call `syncer.Close`, but we need to do this rollback in this scenario. /cc @lance6716